src/components/chatbox/welcome-message.tsx

```diff
@@ -14,6 +14,26 @@ import Text from 'components/text'
 
 import s from './welcome-message.module.css'
 
+// this is an arbitrary, light abstraction to separate
+// the content/copy from the component
+const rightColumn = [
+	{
+		icon: <IconUser16 />,
+		heading: 'Personalized recommendations',
+		text: 'FIX ME', // TODO(kevinwang): fix before launch
+	},
+	{
+		icon: <IconDiscussionCircle16 />,
+		heading: 'Natural language conversations',
+		text: 'FIX ME', // TODO(kevinwang): fix before launch
+	},
+	{
+		icon: <IconBulb16 />,
+		heading: 'Knowledge base',
+		text: 'FIX ME', // TODO(kevinwang): fix before launch
+	},
+]
```

Comment:
@kellyschummer We'll need to update this prior to production rollout

Reply:
@kendallstrautman we can decide if we'd like to add this content to the welcome screen- I've flagged it as a "nice to have" but not a launch blocker.